### PR TITLE
chore: Add `tooltip-placement` example

### PR DESCRIPTION
**Pull request by saideepesh000**

### CODE REVIEW

File: packages/ariakit/src/tooltip/__examples__/tooltip-with-placement/index.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Button } from "ariakit/button";
+import { Tooltip, TooltipAnchor, useTooltipState } from "ariakit/tooltip";
+import "./style.css";
+
+export default function Example() {
+  const tooltip = useTooltipState({ placement: "bottom-end" });
```
Comment (iamsriranga) [inline packages/ariakit/src/tooltip/__examples__/tooltip-with-placement/index.tsx]: @diegohaz , let me know if I should add any other placement examples like `top`, `right`, etc.. or only one example is enough ?

Comment (diegohaz) [reply packages/ariakit/src/tooltip/__examples__/tooltip-with-placement/index.tsx]: One example is fine I guess. :)

**Comment by diegohaz** [comment]
I think we should rename it to just `tooltip-placement` to be less verbose. When we start adding `readme.md` documentation to examples, we can add more descriptive titles like `Tooltip with custom placement` or so (there are other examples with the `with` word that we'll want to change later if we agree on this).

**Comment by saideepesh000** [comment]
> I think we should rename it to just `tooltip-placement` to be less verbose. When we start adding `readme.md` documentation to examples, we can add more descriptive titles like `Tooltip with custom placement` or so (there are other examples with the `with` word that we'll want to change later if we agree on this).

yeah, this makes sense to me

File: packages/ariakit/src/tooltip/__examples__/tooltip-with-placement/index.tsx (hunk 1 above)
Comment (iamsriranga) [reply packages/ariakit/src/tooltip/__examples__/tooltip-with-placement/index.tsx]: made the changes

**Review by diegohaz — APPROVED** [review]
Awesome! Thank you! ❤️

**Pull request merged**